app-backend/sbt

```diff
@@ -20,8 +20,8 @@ declare -r buildProps="project/build.properties"
 
 declare -r sbt_launch_ivy_release_repo="http://repo.typesafe.com/typesafe/ivy-releases"
 declare -r sbt_launch_ivy_snapshot_repo="https://repo.scala-sbt.org/scalasbt/ivy-snapshots"
-declare -r sbt_launch_mvn_release_repo="http://repo.scala-sbt.org/scalasbt/maven-releases"
-declare -r sbt_launch_mvn_snapshot_repo="http://repo.scala-sbt.org/scalasbt/maven-snapshots"
+declare -r sbt_launch_mvn_release_repo="https://repo.scala-sbt.org/scalasbt/maven-releases"
+declare -r sbt_launch_mvn_snapshot_repo="https://repo.scala-sbt.org/scalasbt/maven-snapshots"
```

Comment:
This resolves a failure I was encountering when provisioning a local build environment. I don't think they're hosting anything on port 80 anymore.

Reply:
If you look at the top of this file there is a link to the repo where this is from. It might be sensible to update the whole file to the latest version there. I ran into a similar problem when setting up my new machine.